app/Console/Commands/TweetAboutWeeklySummaryCommand.php

```diff
@@ -25,7 +26,7 @@ public function handle(): int
             return self::SUCCESS;
         }
 
-        Twitter::tweet("There were $streamsCount streams last week. 👏 Thanks to all the streamers and viewers. 🙏🏻\n Find them here: ".route('archive'));
+        Twitter::tweet($streamsCount.Str::plural(' stream', $streamsCount)." happened last week. 👏 Thanks to all the streamers and viewers. 🙏🏻\n Find them here: ".route('archive'));
```

Comment:
I think we can improve it a bit more. Can you please change the text to:
* `Last week, there were 2 streams...`
* `Last week, there was 1 stream...`

Reply:
Agreed, I do prefer your wording. I've made that change and also changed a couple of other parts that didn't make sense when dealing with a single stream, hope you don't mind!

I've swapped it to use `sprintf()` as the line was getting a little too long for my liking, happy to change this back though if you prefer.

As always, let me know if any further changes need making 👍